frontend/src/components/Request.jsx

```diff
@@ -25,6 +25,7 @@ export default function Request() {
 
   const [packageData, setPackageData] = useState(null);
   const [personalCareProducts, setPersonalCareProducts] = useState([]);
+  const [dataOffline, setDataOffline] = useState(false);
```

Comment:
`dataOffline` shouldn't be necessary since we know if the data was fetched based on if `packageData` is null or `personalCareProducts` has length 0